app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md

```diff
@@ -1,12 +1,14 @@
 ---
 title: Prepare for your teacher training interview
-subcategory: Your teacher training application
 description: |-
   Prepare for a teacher training interview. Learn more about what the interview involves, what you might be asked, and how to get help to prepare.
 related_content:
-    Improve your subject knowledge : "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Improve your subject knowledge : "/how-to-apply-for-teacher-training/subject-knowledge-enhancement"
     What teacher training is like : "/train-to-be-a-teacher/initial-teacher-training"
     Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+navigation: 30.20
+navigation_title: Teacher training interviews
+navigation_description: Find out how to prepare for your teacher training interviews.
```

Comment:
@gemmadallmandfe @philspray I've tweaked the nav description so it's 'interview**s**' (plural) but it's not rendering in my review app - is it the same for you two?

Reply:
Yes it's still 'interview' in review here.

Reply:
Title has changed but not the description

Reply:
Fine now - I made a small edit to the maths page (taking a full-stop outside a link) and it seems to have pushed this change through as well